elastic-agent-client.proto

```diff
@@ -112,15 +112,190 @@ message ActionResponse {
     bytes result = 4;
 }
 
+// Type of transaction to start.
+enum StoreTxnType {
+    READ_ONLY = 0;
+    READ_WRITE = 1;
+}
+
+// Begins a new transaction.
+//
+// A started transaction must either have commit or discard called.
+message StoreBeginTxnRequest {
+    // Token that is used to unique identify the connection to the Elastic Agent.
+    string token = 1;
+    // ID of the unit.
+    string unit_id = 2;
+    // Type of transaction to start.
+    StoreTxnType type = 3;
+}
```

Comment:
Risking to sound a bit pedantic, but the comments use _start_ instead of _begin_, wouldn't be better to keep a consistent vocabulary? Therefore to change Begin on the names to Start?

Reply:
Sure I can update it.